tasks/utils/github.js

```diff
@@ -0,0 +1,70 @@
+const axios = require("axios")
+
+export const createGist = async (githubApiToken, encryptedOffchainSecrets) => {
+  const content = JSON.stringify(encryptedOffchainSecrets)
+
+  const headers = {
+    Authorization: `token ${githubApiToken}`,
+  }
+
+  // construct the API endpoint for creating a Gist
+  const url = "https://api.github.com/gists"
+  const body = {
+    public: true,
```

Comment:
```suggestion
    public: false,
```
We want private gists